PowerShellBestPractices.md

```diff
@@ -0,0 +1,103 @@
+#PowerShell Best Practices
+
```

Comment:
From today's discussions pls organize the documentation as follows:

1) Best practices for Scripts/DSC ...
2) Sort rules by Severity. - for rules that have no sev assigned, create "open" bucket
3) Hyperlink to existing rules, tag rules with no documentation/existing rule to issues